internal/bminventory/inventory.go

```diff
@@ -172,7 +172,7 @@ const ignitionConfigFormat = `{
         {
                 "name": "configure-static-ip.service",
                 "enabled": true,
-                "contents": "[Unit]\nDescription=Static IP Configuration\nBefore=NetworkManager.service\nDefaultDependencies=no\n[Service]\nUser=root\nType=oneshot\nTimeoutSec=10\nExecStart=/bin/bash /var/tmp/configure-static-ip.sh\nRemainAfterExit=no\n[Install]\nWantedBy=multi-user.target"
+                "contents": "[Unit]\nDescription=Static IP Configuration\nBefore=NetworkManager.service\nDefaultDependencies=no\n[Service]\nUser=root\nType=oneshot\nTimeoutSec=10\nExecStart=/bin/bash /usr/local/bin/configure-static-ip.sh\nPrivateTmp=yes\nRemainAfterExit=no\n[Install]\nWantedBy=multi-user.target"
```

Comment:
Do you need `PrivateTmp=yes` even after moving the file to `/etc`? I'm not sure I understand why this argument is needed.

Reply:
PrivateTemp means that a directory created under /var/tmp , named <service-uuid> and is mapped to the service as /var/tmp. This way the data in it is protected from /var/tmp cleanup services. We are using this directory for template file